File: w09/d01/enoch_riese/mail/spec/mail_spec.rb

require_relative "spec_helper"
require_relative "../lib/mail_item"

describe MailItem do
  it "should have a ship_to_address" do
    mail = MailItem.new({ship_to_address: "140 Franklin St."})
    expect(mail.ship_to_address).to be
  end
  it "should have a stamp_value" do
    mail = MailItem.new({stamp_value: 0.37})
    expect(mail.stamp_value).to be
  end
end

File: w09/d03/jaclyn_jimenez/mail_app/spec/javascripts/mail_spec.js
// MailItem should have a shipToAddress string
// MailItem should have a stampValue number

describe("a mail item", function(){
  var mail;
  beforeEach(function(){
    mail = new MailItem();
  });

  it("should have a shipToAddress string", function() {
    expect(mail.shipToAddress).toMatch("902 Broadway");
  });

  it("should have a stampValue number", function() {
    expect(mail.stampValue).toMatch(20);
  });


});
